subiquity/server/controllers/install.py

```diff
@@ -732,6 +732,13 @@ async def postinstall(self, *, context):
 
         fs_controller = self.app.controllers.Filesystem
         if fs_controller.use_snapd_install_api():
+            if fs_controller.use_tpm:
+                # This will generate a recovery key which will initially expire
+                # after a very short duration (e.g., 5 minutes).
+                # We need to reach the finish_install step within that timeframe,
+                # otherwise the key will be considered expired. So let's keep
+                # the two calls next to one another.
+                await fs_controller.fetch_core_boot_recovery_key()
```

Comment:
This all looks fine but I was expecting to see the existing `fs_controller.fetch_core_boot_recovery_key()` call removed, I assume we want to remove the other one?